project.clj

```diff
@@ -19,21 +22,20 @@
   :plugins [[thomasa/mranderson "0.4.8"]]
   :filespecs [{:type :bytes :path "refactor-nrepl/refactor-nrepl/project.clj" :bytes ~(slurp "project.clj")}]
   :profiles {:provided {:dependencies [[cider/cider-nrepl "0.14.0"]
-                                       [org.clojure/clojure "1.7.0"]]}
+                                       [org.clojure/clojure "1.8.0"]]}
              :test {:dependencies [[print-foo "1.0.2"]]
                     :src-paths ["test/resources"]}
-             :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]]}
              :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]
                                   [org.clojure/clojurescript "1.9.908"]]}
              :dev {:plugins [[jonase/eastwood "0.2.0"]]
                    :global-vars {*warn-on-reflection* true}
-                   :dependencies [[org.clojure/clojurescript "1.9.89"]
-                                  [com.cemerick/piggieback "0.2.2"]
+                   :dependencies [[org.clojure/clojurescript "1.10.238"]
+                                  [cider/piggieback "0.2.3"]
```

Comment:
That should 0.3.6.

Reply:
and `cemerick.piggieback` needs to be updated in `refactor-nrepl.ns.resolve-missing-test`

there are still failures and errors when tests are run but it compiles after the above changes